app/controllers/api/v1/jwt_api/auth_user_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    module JwtApi
+      class AuthUserController < JwtApiController
+        before_action :set_person, only: %i[create]
+
+        def create
+          render json: current_auth_user, serializer: Api::AuthUserSerializer
+        end
+
+        private
+
+        def set_person
+          return if current_auth_user&.person.present?
+
+          result = { result: current_auth_user.to_json }
+          render(status: :not_found, json: result)
+        end
```

Comment:
Compare `app/controllers/api/v1/jwt_api/protocol_subscriptions_controller.rb` to this file. Here, you explicitly check if a current_auth_user has a person, and then render the method. There, you just return current_user (which in this case is the person of the current_auth user).

I think that the JwtApiController should implement a before filter that authenticates the user (i.e., it calls auth_user or authenticate_auth_user, or whatever is needed to trigger the authentication method), and then also checks that that the current_auth_user has a person (either with current_user or current_auth_user.person).

So then we no longer need to check whether a current_auth_user.person exists in any deriving subclass like we do here, and can no longer forget to check (like we do in `app/controllers/api/v1/jwt_api/protocol_subscriptions_controller.rb`).